### PR TITLE
Native Notifications

### PR DESCRIPTION
**Pull request by emmauss**
Simple native notification implementation. Currently supports Android, Linux and Windows.
Windows apps would need to be packaged to work.

### CODE REVIEW

File: src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.Windows/WinRT/Windows.Ui.Notifications.Generated.cs [hunk 1]
```diff
@@ -0,0 +1,3612 @@
+#pragma warning disable 108
```
Comment (maxkatz6) [inline src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.Windows/WinRT/Windows.Ui.Notifications.Generated.cs:1]: This file doesn't need to be generated.
You also can use SourceGen for MicroCom:
- https://www.nuget.org/packages/MicroCom.CodeGenerator.Roslyn
- rename ".idl" to ".mcidl"
- Include it with [AdditionalFiles](https://github.com/dotnet/roslyn/blob/main/docs/analyzers/Using%20Additional%20Files.md)

File: src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.Android/Avalonia.Labs.Notifications.Android.csproj [hunk 2]
```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net8.0-android</TargetFramework>
```
Comment (maxkatz6) [inline src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.Android/Avalonia.Labs.Notifications.Android.csproj:3]: Why won't we have it as a single multi-target project? Should be simple enough until each target won't have unnecessary dependencies.

Comment (emmauss) [reply src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.Android/Avalonia.Labs.Notifications.Android.csproj:3]: I'll update the projects.

File: src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.Windows/WinRT/Windows.Ui.Notifications.Generated.cs (hunk 1 above)
Comment (emmauss) [reply src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.Windows/WinRT/Windows.Ui.Notifications.Generated.cs]: Done.

File: src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.Android/Avalonia.Labs.Notifications.Android.csproj (hunk 2 above)
Comment (emmauss) [reply src/Avalonia.Labs.Notifications/Avalonia.Labs.Notifications.Android/Avalonia.Labs.Notifications.Android.csproj]: They are all merged now.

**Comment by affederaffe** [comment]
The Linux implementation may use DBus instead of pulling in that many native dependencies, see https://specifications.freedesktop.org/notification-spec/notification-spec-latest.html.
If it's desired, I could open a PR for that

**Comment by emmauss** [comment]
> The Linux implementation may use DBus instead of pulling in that many native dependencies, see https://specifications.freedesktop.org/notification-spec/notification-spec-latest.html. If it's desired, I could open a PR for that

I would appreciate it if you could. I'm not familiar with dbus and its protocol.

**Pull request merged**